### PR TITLE
Direct users to GitHub discussions page for usage questions

### PR DESCRIPTION
**Pull request by andersy005**
Towards https://github.com/pydata/xarray/issues/4549

 - [ ] Closes #xxxx
 - [ ] Tests added
 - [x] Passes `isort . && black . && mypy . && flake8`
 - [ ] User visible changes (including notable bug fixes) are documented in `whats-new.rst`
 - [ ] New functions/methods are listed in `api.rst`

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
...
+blank_issues_enabled: false
 contact_links:
   - name: Usage question
-    url: https://stackoverflow.com/questions/tagged/python-xarray
-    about: "Post a question on Stack Overflow using the #python-xarray
-    tag. These are regularly reviewed by xarray's maintainers, and questions which
-    include a reproducible example will receive a response."
\ No newline at end of file
+    url: https://github.com/pydata/xarray/discussions
+    about: Ask questions and discuss with other community members here.
```
Comment (max-sixty) [inline .github/ISSUE_TEMPLATE/config.yml:5]: Should we specify the reproducible examples here, or is there a separate template for discussions?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
-    url: https://stackoverflow.com/questions/tagged/python-xarray
-    about: "Post a question on Stack Overflow using the #python-xarray
-    tag. These are regularly reviewed by xarray's maintainers, and questions which
-    include a reproducible example will receive a response."
\ No newline at end of file
+    url: https://github.com/pydata/xarray/discussions
+    about: |
+      Ask questions and discuss with other community members here.
+      If you have a question like `How do I concatenate a list of datasets?` then 
+      please include a self-contained reproducible example if possible.
```
Comment (andersy005) [inline .github/ISSUE_TEMPLATE/config.yml:8]: @max-sixty, let me know if this enough and/or whether this is what you are suggesting in 

> Should we specify the reproducible examples here, or is there a separate template for discussions?

Comment (max-sixty) [reply .github/ISSUE_TEMPLATE/config.yml:8]: Yes great! I'm not sure if we can fit the link we generally use <https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports>, so probably we go with this and add the link when we can have a standard discussion template.

Thanks!

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
   - name: Usage question
-    url: https://stackoverflow.com/questions/tagged/python-xarray
-    about: "Post a question on Stack Overflow using the #python-xarray
-    tag. These are regularly reviewed by xarray's maintainers, and questions which
-    include a reproducible example will receive a response."
\ No newline at end of file
+    url: https://github.com/pydata/xarray/discussions
+    about: |
+      Ask questions and discuss with other community members here.
+      If you have a question like `How do I concatenate a list of datasets?` then
```
Comment (keewis) [inline .github/ISSUE_TEMPLATE/config.yml:7]: it seems `about` is a plain text description, so we unfortunately can't use markdown here:
```suggestion
      If you have a question like "How do I concatenate a list of datasets?" then
```

**Review by keewis — APPROVED** [review]
thanks, @andersy005, looks good to me. I'd change the backticks (see below), but other than that I think we can merge (and add the "reproducible example" link once discussion templates become available)

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (andersy005) [reply .github/ISSUE_TEMPLATE/config.yml]: 👍🏽

**Comment by keewis** [comment]
thanks for the quick update, @andersy005. If there are no further comments: let's merge?

**Comment by andersy005** [comment]
> thanks for the quick update, @andersy005. If there are no further comments: let's merge?

I think this is ready to be merged...

**Comment by keewis** [comment]
oh, yes, I just wanted to give others the opportunity to comment before merging.

**Pull request merged**